components/layout/index.styl

```diff
@@ -1,12 +1,13 @@
-@import '../../node_modules/nib'
+// Common (includes Nib)
+@import '../../node_modules/artsy-elan/components/lib'
+
+// Components
+@import '../../node_modules/artsy-elan/assets/stylesheets/components'
+@import '../nav'
 @import '../header'
+
 @import '../footer'
 @import '../body'
-@import '../carousel/carousel.styl'
-@import '../carousel/desktop_carousel.styl'
-@import 'button'
-@import '../header/desktop'
+@import '../carousel/carousel'
+@import '../carousel/desktop_carousel'
```

Comment:
@dzucconi @xtina-starr started this project mobile-first and then overrode desktop. The separate stylesheets was my idea, thought it might make it easier to keep these things in more discrete chunks, but you're right, it could go the other way as well.